english/make-friends/where_am_i

```diff
@@ -0,0 +1,16 @@
+You realise you need as much information as you need to get out of this situation.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.